otter/test/test_util.py

```diff
@@ -892,3 +893,24 @@ def test_auth_error_propogates(self):
         d = retry_on_unauth(func, auth)
         self.failureResultOf(d, ValueError)
         auth.assert_called_once_with()
+
+
+class TextTests(SynchronousTestCase):
+    """
+    Tests for :func:`text`
+    """
+
+    def test_unicode(self):
+        """ unicode is returned as such """
+        self.assertEqual(text(u"test"), u"test")
+
+    def test_string(self):
+        """ string is converted to unicode """
+        self.assertEqual(text("test"), u"test")
+
+    def test_py3(self):
+        """ Not implemented for PY3 """
```

Comment:
This should be a link to an issue.  Patching `six.PY2` is a pretty gross way to test this :-\\